Base/usr/share/man/man1/su.md

```diff
@@ -1,41 +1,42 @@
 ## Name
 
-su - run a command with substitute user
+su - switch to another user
 
 ## Synopsis
 
-```
-$ su [user]
+```**sh
+$ su [--help] [--version] <user>
```

Comment:
Help and version don't need to be here in the manpage, you'll see that when running the app.

Reply:
Should Help and version needs to be present in the Options part for su? @kleinesfilmroellchen

Reply:
No, they should not.

Reply:
> No, they should not.

They are already present in 34 other man pages, so I'm not too sure about that,